lib/backend/syncersv1/updateprocessors/networkpolicyprocessor.go

```diff
@@ -51,6 +51,7 @@ func convertNetworkPolicyV2ToV1Value(val interface{}) (interface{}, error) {
 
 	if v3res.Namespace != "" {
 		nsSelector := fmt.Sprintf("%s == '%s'", apiv3.LabelNamespace, v3res.Namespace)
+		nsSelector = strings.Replace(nsSelector, "all()", "has(projectcalico.org/namespace)", -1)
```

Comment:
I think this line actually isn't doing anything - `nsSelector` will never be `all()` at this point, since we define it one line above :)

Reply:
Fixed